src/shared/components/BaseToastMessages.tsx

```diff
@@ -128,25 +90,22 @@ export default function BaseToastMessages({
       >
         {messages.map(message => (
           <li
-            className={classnames(
-              'relative w-full container hover:cursor-pointer',
```

Comment:
This change was to remove the `hover:cursor-pointer` styling. Get that for free with the `raised` variant of `Callout`.